wordpress-mobile/WordPress-Aztec-iOS
# Standardize check for formatting

Instead of using multiple methods to check for formatters active, this PR switch the approach to use a data driven approach where the mapping of formatters and typing attributes is done using a dictionary.

How to test:
 - Start the demo app
 - Navigate around and see if the toolbar is updated correctly.
 - Apply style to new ranges and location and see if the toolbar updates correctly.
